src/reportWebVitals.js

```diff
@@ -1,16 +1,15 @@
 const reportWebVitals = (onPerfEntry) => {
-    if (onPerfEntry && onPerfEntry instanceof Function) {
-      import('web-vitals').then(({
-        getCLS, getFID, getFCP, getLCP, getTTFB,
-      }) => {
-        getCLS(onPerfEntry);
-        getFID(onPerfEntry);
-        getFCP(onPerfEntry);
-        getLCP(onPerfEntry);
-        getTTFB(onPerfEntry);
-      });
-    }
-  };
-  
-  export default reportWebVitals;
-  
\ No newline at end of file
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```

Comment:
- [ ] [Optional]remove this file as it is part of the boilerplate created from create react app